README.md

```diff
@@ -101,33 +88,29 @@ ramaze                    1936.36       10.44%
 ```
 Framework       Allocs/Req Memsize/Req
 --------------------------------------
-rack                    38        3256
-syro                    43        3536
-hobbit                  45        3808
-roda                    45        3752
-mustermann              51        4112
-cuba                    53        4072
-rack-response           56        5312
-brooklyn                69        6412
-nancy                   75        7644
-nyny                    82        8396
-rackstep                82        6008
-rambutan                85        6848
-hanami-router           90        7859
-rails                  113       10311
-gin                    198       17219
-sinatra                212       15783
-grape                  257       20236
-ramaze                 464       41360
+rack                    34        2640
+syro                    38        3328
+roda                    38        3328
+cuba                    40        3408
+hobbit                  42        3648
+rack-response           49        3856
+rails-metal             53        5448
+hanami-router           61        4600
+rackstep                74        4664
+rambutan                76        5664
+rack-app                76        6904
+flame                   96        7130
+sinatra                182       12008
+grape                  237       17168
+rails-api              378       27576
 ```
 <!-- mem_table_end -->
 
 
-These numbers were collected on:
+Test environment:
 
-- Ubuntu 15.10 64-bit (kernel: 4.2.0-35-generic)
```

Comment:
Do not update hardware specs, as the hardware used in your end does not match mine, which aligns with my request about performance numbers listed above.